.changeset/nervous-yaks-destroy.md

```diff
@@ -0,0 +1,5 @@
+---
+"eslint-plugin-regexp": patch
```

Comment:
I don't think this is a patch. Probably major because it changes the recommended configuration.

```suggestion
"eslint-plugin-regexp": major
```